ChatQnA/tests/_test_chatqna_pinecone_on_xeon.sh

```diff
@@ -0,0 +1,232 @@
+#!/bin/bash
```

Comment:
'_' before test means a test is not ready for test, so please remove it from the script name.

Reply:
And we refine the test structure recent days, please follow this one to make the changes, https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/tests/test_chatqna_on_xeon.sh.

Reply:
updated the tests as per the above suggestion

Reply:
Why duplicate the test scripts, you should remove this one.